x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/overview/metrics/container_metrics.tsx

```diff
@@ -20,20 +20,22 @@ interface Props {
 }
 
 export const ContainerMetrics = (props: Props) => {
-  const isK8sContainer = useIntegrationCheck({ dependsOn: INTEGRATIONS.kubernetesContainer });
+  const isDockerContainer = useIntegrationCheck({ dependsOn: INTEGRATIONS.docker });
```

Comment:
Same here: It wasn't perfect to assume that docker is the "default" integration either if the K8s is not available but maybe that's not a valid concern for now.